autopilot/contractor.go

```diff
@@ -510,7 +513,7 @@ func (c *contractor) performWalletMaintenance(ctx context.Context) error {
 			severity = alerts.SeverityWarning
 		}
 
-		c.ap.alerts.Register(alerts.Alert{
+		err = c.ap.alerts.RegisterAlert(ctx, alerts.Alert{
```

Comment:
Now that we've centralised alerts, should we make the "origin" a required property? I know we can pass it in the `Data` but I think it makes sense to make it required so the UI knows whether the alert is a `worker` alert or if it originated from the autopilot?